src/logic/PlayerController.cpp

```diff
@@ -835,8 +835,7 @@ void PlayerController::onUpdateByInput(float deltaTime)
             }
             else
             {
-                // This happens when NPC enters area that should not be accessible
```

Comment:
Well, we will have to debug this one way or another, for now this logging functionality doesn't help at all because it literally spams junk even though NPC stands on valid surface